CHANGELOG.md

```diff
@@ -12,16 +12,18 @@
 
 ## Improvements
 
-# v1.28.0-rc1 / 2024-07-01
+# v1.28.0-rc2 / 2024-07-04
 
-This is the first release candidate of the upcoming MANDATORY Lotus v1.28.0 release, which will deliver the Filecoin network version 23, codenamed Waffle 🧇.
+This is the second release candidate of the upcoming MANDATORY Lotus v1.28.0 release, which will deliver the Filecoin network version 23, codenamed Waffle 🧇.
 
 **This release canidate does NOT set a calibration network upgrade epoch, it will be added in the second release candidate, expected to be released July 4th. This release candidate does NOT set the mainnet upgrade epoch yet, which will be updated in the final release.**
```

Comment:
does NOT?

Reply:
Addressed in: https://github.com/filecoin-project/lotus/pull/12179/commits/18f961afe43552698824215be3509a65292fe25d